cli/cmd/plugin/unmanaged-cluster/cmd/create.go

```diff
@@ -106,7 +106,7 @@ func create(cmd *cobra.Command, args []string) error {
 	tm := tanzu.New(log)
 	err = tm.Deploy(clusterConfig)
 	if err != nil {
-		log.Error(err.Error())
+		log.Errorf("%s\n", err.Error())
```

Comment:
An alternative here would be to change the `log.Error` method to:

```diff
+ fmt.Println(message)
- fmt.Print(message)
```

but I'm also good with your change

Reply:
let's revisit this option as an enhancement to the log package.